tests/test_https.py

```diff
@@ -75,7 +75,8 @@ def test_not_trusted_ca():
     not hasattr(tests.ssl_context(), "minimum_version"),
     reason="ssl doesn't support TLS min/max",
 )
-def test_set_min_tls_version():
+@pytest.mark.parametrize("version", (None, ssl.TLSVersion.TLSv1_2, "TLSv1_2"))
+def test_set_min_tls_version(version):
```

Comment:
line 84 should use `version` then

Reply:
Yep, also remove the old max testing too to align. All done in next changeset.